src/content/quickstart/openshift/_index.md

```diff
@@ -35,17 +36,21 @@ customize the AWS instance type as shown below.
 
 {{< include "quickstart/openshift/run_prep_for_subm.md" >}}
 
-### Install `subctl`
+### Install subctl
```

Comment:
You don't think we should use `code style` in headings?

Reply:
Fixed